tutum.yml

```diff
@@ -0,0 +1,21 @@
+parse:
+  image: didierfranc/parse-server
```

Comment:
I made this image. Now I added the Dockerfile to the repo it's easy to make one : 

```
git clone https://github.com/ParsePlatform/parse-server-example.git
docker build -t username/parse-server .
docker push username/parseserver
```
I'm maintaining this image to stay updated, is it a problem ? I asked [here](https://github.com/ParsePlatform/parse-server/issues/62) to have an official image.